packages/components/src/placeholder/index.js

```diff
@@ -34,6 +34,7 @@ function Placeholder( { icon, children, label, instructions, className, notices,
 			<div className={ fieldsetClasses }>
 				{ children }
 			</div>
+			{ !! append && <div className="components-placeholder__append">{ append }</div> }
```

Comment:
Instead of adding an append prop can't we pass what is going to appended as part of the children (last child)? If it is style purposes we can wrap the current children in a div and then pass another div with our appender class.

Reply:
I'm not really sure I understand what you're saying?

Reply:
Hi @senadir the Placeholder component already includes a mechanism for appending (everything added as a child is rendered inside the component) so having an append prop there seems redundant.
The problem was that MediaPlaceholder did not pass the children to the Placeholder component, but we can quickly solve that.
I created this commit to more easily explain what I'm suggesting https://github.com/WordPress/gutenberg/commit/a44eda82f2312abea613c9719acc6bb2130c579e. In this commit I add the color palette in the placeholder without using an append prop, I pass it as a child. With this change, we can then easily avoid the append addition.